media-sound/amc/amc-1.14.0.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A'dam Music Composer"
+DESCRIPTION="This Mini-Scheme Interpreter is based on SCHEME Interpreter in \
+Common Lisp in Appendix of T.Matsuda & K.Saigo, Programming of LISP, \
+archive No5 (1987) p6 - p42 (published in Japan)."
+HOMEPAGE="https://github.com/HaikuArchives/AMC"
+COPYRIGHT="1989 Atsushi Moriwaki "
+LICENSE="Public Domain"
+REVISION="1"
+srcGitRev="451087418bc91f626ba7cb1b112816e3bac898f9"
+SOURCE_URI="https://github.com/HaikuArchives/AMC/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="8e90759b128c66f3a3443fecf0b9ffda112a911be77fc14d7666d7bb051b98f7"
+SOURCE_DIR="AMC-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2"
+
+PROVIDES="
+	amc = $portVersion
+	app:AMC = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc
+	"
+
+BUILD()
+{
+	cd amc/src
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd amc/src
+	mkdir -p $appsDir/AMC
+	cp -a amc $appsDir/AMC/A\'dams\ Music\ Composer
```

Comment:
I don't want to open that can of worms, either. :)
So, not talking in general, but only for this case, here's my rational for putting the docs and tunes with the binary into a subfolder in $appsDir: users easily find the app in its folder in $appsDir and thereby also find its docs and the example tunes.  Since there's no help menu in the app itself, the users otherwise probably won't know that AMC comes with docs and example tunes and where to look for them.

I assumed the declaration of "app:" under  PROVIDES had to be the exact name of the app under that it will be installed. Guess that was wrong. :)
To me the app's filename should be the name that appears in the app's window title. In this case it's "A'dams Music Composer".
Hmm... I stand corrected, here it's "the A'dams Music Composer". Man, this app is sucha mess... I dare you to look at its source... :)